R/read.R

```diff
@@ -289,23 +309,89 @@ readH5AD <- function(file, X_name = NULL, use_hdf5 = FALSE,
     mat
 }
 
+.convert_element <- function(obj, path, file, recursive=FALSE) {
+    # Should we wrap this in suppressWarnings? rhdf5 will warn that it
+    # can't yet read enum (factor/boolean) in attributes
+    element_attrs <- rhdf5::h5readAttributes(file, path)
```

Comment:
Hmmm...maybe. If it's something we handle so the warning isn't meaningful to the user then I guess that would make sense. There's just the risk that we suppress a future warning that it would be useful to know about.